gitgud/levels/reworking/__init__.py

```diff
@@ -0,0 +1,8 @@
+from collections import OrderedDict
+
+import pkg_resources
+
+from gitgud.levels.util import BasicChallenge
+
+all_challenges = OrderedDict()
+all_challenges['octopus'] = BasicChallenge('octopus', pkg_resources.resource_filename(__name__, '_octopus/'))
```

Comment:
You'll need to replace all occurrences of  `octopus` with `cherrypick` here. `_cherrypick` is what points to the folders that actually contain your code, and the other occurances are the name that it is registered by